UnitsNet/Scripts/UnitDefinitions/MassFlow.json

```diff
@@ -0,0 +1,31 @@
+{
+    "Name": "MassFlow",
+    "BaseUnit": "KilogramPerSecond",
+    "XmlDoc": "",
+    "Units": [
+        {
+            "SingularName": "KilogramPerSecond",
+            "PluralName": "KilogramsPerSecond",
+            "FromUnitToBaseFunc": "x",
+            "FromBaseToUnitFunc": "x",
+            "Localization": [
+                {
+                    "Culture": "en-US",
+                    "Abbreviations": ["kg/s"]
+                }
+            ]
+        },
+		{
+            "SingularName": "TonnePerDay",
+            "PluralName": "TonnesPerDay",
+            "FromUnitToBaseFunc": "x*86.4000",
+            "FromBaseToUnitFunc": "x/86.4000",
```

Comment:
I'll fix this tonight or tomorrow morning when I have access to VS.

There are an awful lot of constants. I'm considering checking the complete diff once more, it's so easy to invert numbers like this. Do you think I should do that?